doc/api/fs.md

```diff
@@ -70,6 +70,12 @@ the entire process until they complete--halting all connections.
 The relative path to a filename can be used. Remember, however, that this path
 will be relative to `process.cwd()`.
 
+For most `fs` module functions, the `path` or `filename` argument may be passed
+as a WHATWG [URL][] object. Only [URL][] objects using the `file:` protocol are
+supported. Use of [URL][] objects is currently still experimental.
```

Comment:
I second that. Examples may be helpful. Will work on that tomorrow. Thanks a lot !